### ISSUE TITLE
does our Docker-ized fuskei only work inside of incognito browser windows?

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#1694]

**Comment by eecavanna** [comment]
I will try to answer the question in the issue name.

**Comment by eecavanna** [comment]
No, it works in both Incognito and non-Incognito browser windows.

<img width="977" alt="image" src="https://github.com/microbiomedata/nmdc-schema/assets/134325062/fff0d880-d336-4483-a8e4-be824143244a">

Closing.

**Comment by turbomam** [comment]
Sorry, I should have kept this up to date. Fuseki, from Docker, does not run in a normal browser session on either my Ubuntu workstation or in my Intel LBL MacBook Pro. I believe I have tried both chrome and Firefox. I can provide screen shots, or we could just Zoom.

**Comment by eecavanna** [comment]
Will you elaborate on what you mean by "does not run"? What does the web browser show (e.g. blank page, the login prompt, an error message, etc.)?

**Comment by turbomam** [comment]
Sorry about that. Totally blank browser window, plus warning about unsafe URL in navigation bar.

**Comment by eecavanna** [comment]
Thanks. Interesting. 

What happens when you visit other `http://` URLs in those web browsers (specifically, those that don't automatically redirect to an `https://` URL)?

Here's an example of such a URL: 
- http://web.simmons.edu/~grabiner/comm244/weektwo/links.html

If that example web page loads OK for you, please confirm that your web browser hasn't automatically navigated to the `https://` version of it.

**Comment by eecavanna** [comment]
> I can provide screen shots

I would like to see a screenshot of the error message that also shows the URL in the nagivation bar. That will show me the error message and confirm that the web browser didn't automatically try to go the `https://` version of the URL.

**Comment by eecavanna** [comment]
Moved to next sprint. I expect it to take under an hour to resolve.

**Comment by eecavanna** [comment]
Hi @turbomam, will you take a look at comments https://github.com/microbiomedata/nmdc-schema/issues/1694#issuecomment-1911039419 and https://github.com/microbiomedata/nmdc-schema/issues/1694#issuecomment-1911098139 and tell me what you think?

**Comment by eecavanna** [comment]
@turbomam, is this still happening on your Ubuntu machine? If so, will you provide more info about the behavior (see comments https://github.com/microbiomedata/nmdc-schema/issues/1694#issuecomment-1911039419 and https://github.com/microbiomedata/nmdc-schema/issues/1694#issuecomment-1911098139)?

**Comment by eecavanna** [comment]
I'm in favor of moving this to the next sprint if there are no updates by Thursday night.

**Comment by eecavanna** [comment]
I don't have the bandwidth to work on this this sprint. I'll move it to the next one.

**Comment by eecavanna** [comment]
On second thought, I will reassign to @turbomam to provide more information.

**Comment by ssarrafan** [comment]
I will remove from the sprint. @turbomam @eecavanna if still active please add to the next sprint.

**Comment by eecavanna** [comment]
@turbomam and I looked into this together today.

For reference, he recently reformatted his Linux machine, so the context may be different from what it was when this issue was created.

Today, he demonstrated that the Fuseki container would not start up properly. It reported that it didn't have permission to do something in `/fuseki/fuseki-data` (a filesystem path in the container). For reference, that is mapped to `./local/fuseki-data` on his host machine.

He checked the permissions of `./local/fuseki-data` on his host machine and found that they were:
```console
$ stat local/fuseki-data/
# ...
Access: (0755/drwxr-xr-x)  Uid: ( 1000/    mark)   Gid: ( 1000/    mark)
# ...
```

As an experiment, he ran `$ chmod -R 777 local/fuseki-data/`, then restarted the Docker stack. Now, the Fuseki container started up normally.

With the Fuseki container running normally, he was able to access its web GUI from both a normal and an incognito web browser.

The only issue I see as outstanding here is that the container didn't start up normally until after he loosen the directory permissions so much. I don't want Linux developers to have to do that.

**Comment by turbomam** [comment]
Closing as not planned. The team is moving away from SPARQL/triplestore/RDF infrastructure in nmdc-schema (tracked in https://github.com/microbiomedata/nmdc-schema/issues/2970).
